packages/libs/react-ui/src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  containerClass,
+  disabledClass,
+  outlinedClass,
+  textAreaClass,
+  textAreaContainerClass,
+} from './TextArea.css';
+
+import type { Sprinkles } from '@theme/sprinkles.css';
+import { sprinkles } from '@theme/sprinkles.css';
+import classNames from 'classnames';
+import type { ChangeEvent, FC, TextareaHTMLAttributes } from 'react';
+import React, { forwardRef } from 'react';
+
+export interface ITextareaProps
+  extends Omit<
+      TextareaHTMLAttributes<HTMLTextAreaElement>,
+      'as' | 'disabled' | 'children' | 'className' | 'id'
+    >,
+    Partial<Pick<Sprinkles, 'fontFamily'>> {
+  id: string;
+  value: string;
+  onValueChange: (event: ChangeEvent<HTMLTextAreaElement>) => void;
```

Comment:
I believe onChange will be included in the TextareaHTMLAttributes so you shouldn't need to add it here or on the textarea element, it should be part of `rest`